package.json

```diff
@@ -2,6 +2,7 @@
   "name": "in-view",
   "version": "0.4.2",
   "description": "Get notified when a DOM element enters or exits the viewport.",
+  "repository": "https://github.com/camwiegert/in-view",
```

Comment:
Don't we want this in a slight different format? Per [the docs](https://docs.npmjs.com/files/package.json#repository):

```json
"repository": {
    "type": "git",
    "url": "https://github.com/camwiegert/in-view.git"
}
```